### PR TITLE
docs: translations for pt-br

### PR DESCRIPTION
**Pull request by Rogerio-Viana**
# Summary

Translate docs files of uk to pt-br

Files not translate:
commands asdf - help.txt file
create plugin - plugins-create.md file
plugins list - plugins-all (asdf-vm/asdf-plugins/master/README.md)

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -1,86 +1,79 @@
...
+                EditOnGithubPlugin.create(
+                    "https://github.com/asdf-vm/asdf/blob/master/docs/", {
+                        customURLs: {
+                            "https://raw.githubusercontent.com/asdf-vm/asdf-plugins/master/README.md": "https://github.com/asdf-vm/asdf-plugins/edit/master/README.md",
+                            "https://raw.githubusercontent.com/asdf-vm/asdf/master/CHANGELOG.md": "https://github.com/asdf-vm/asdf/edit/master/CHANGELOG.md",
+                        },
+                    }
+                ),
+            ],
+        };
```
Comment (Stratus3D) [inline docs/index.html:76]: Why were these lines changed? I think I prefer to the two space indentation.

File: docs/plugins-create.md [hunk 2]
```diff
@@ -55,18 +55,18 @@ The install script should exit with a status of `0` when the installation is suc
...
 
-If you want your plugin to work with asdf version 0.7.* and earlier and version 0.8.* and newer check for the presence of the `ASDF_DOWNLOAD_PATH` environment variable. If it is not set download the source code in the bin/install callback. If it is set assume the `bin/download` script already downloaded it.
+If you want your plugin to work with asdf version 0.7._ and earlier and version 0.8._ and newer check for the presence of the `ASDF_DOWNLOAD_PATH` environment variable. If it is not set download the source code in the bin/install callback. If it is set assume the `bin/download` script already downloaded it.
```
Comment (Stratus3D) [inline docs/plugins-create.md:58]: Why was `*` changed to `_` here? Star is typically recognized as a wildcard character.

File: docs/plugins-create.md [hunk 3]
```diff
@@ -204,7 +204,7 @@ asdf plugin test <plugin-name> <plugin-url> [--asdf-tool-version <version>] [--a
 ```
 
 Only the two first arguments are required.
-If __version_ is specified, the tool will be installed with that specific version. Defaults to whatever returns `asdf latest <plugin-name>`.
+If \__version_ is specified, the tool will be installed with that specific version. Defaults to whatever returns `asdf latest <plugin-name>`.
```
Comment (Stratus3D) [inline docs/plugins-create.md:207]: What is going on with the `\_` here?

File: docs/scripts/docsify-edit-on-github.js [hunk 4]
```diff
@@ -1,47 +1,48 @@
...
-      ].join('')
-
-      hook.afterEach(function (html) {
-        return header + html
-      })
-    }
-  }
-
-  win.EditOnGithubPlugin.create = create
-}) (window)
```
Comment (Stratus3D) [inline docs/scripts/docsify-edit-on-github.js:47]: Why was this code changed?

**Comment by Stratus3D** [comment]
Thanks for the PR @Rogerio-Viana ! I left some comments about some changes on the existing code, I don't understand why they were needed.

@asdf-vm/core do we have anyone that speaks Brazilian Portuguese that could review this?

File: docs/index.html (hunk 1 above)
Comment (Rogerio-Viana) [reply docs/index.html]: I making fixes in format in this file.
please review files
thanks for marking the errors

File: docs/plugins-create.md (hunk 2 above)
Comment (Rogerio-Viana) [reply docs/plugins-create.md]: I making fixes in format in this file.
please review files
thanks for marking the errors

File: docs/plugins-create.md (hunk 3 above)
Comment (Rogerio-Viana) [reply docs/plugins-create.md]: I making fixes in format in this file.
please review files
thanks for marking the errors

File: docs/scripts/docsify-edit-on-github.js (hunk 4 above)
Comment (Rogerio-Viana) [reply docs/scripts/docsify-edit-on-github.js]: I making fixes in format in this file.
please review files
thanks for marking the errors

**Comment by jthegedus** [comment]
I do not speak the language. Best I can do is copy/paste into Google Translate

**Comment by Stratus3D** [comment]
> I do not speak the language. Best I can do is copy/paste into Google Translate

I just pasted a few paragraphs into Google Translate and it looks good. Unless we have someone who can review all the Portuguese I think it is good to merge this.

**Review by Stratus3D — APPROVED** [review]

**Comment by jthegedus** [comment]
The quality of translations is always tied to how active people of the particular language use the translations, only exception is using a paid translation service, and I do not think we're there yet.

**Comment by Rogerio-Viana** [comment]
Thanks for the team. I'm from Brazil and I'm happy to be able to contribute to the project.
